libraries/chain/db_block.cpp

```diff
@@ -103,6 +140,32 @@ std::vector<block_id_type> database::get_block_ids_on_fork(block_id_type head_of
   result.emplace_back(branches.first.back()->previous_id());
   return result;
 }
+    
+void database::check_tansaction_for_duplicated_operations(const signed_transaction& trx)
```

Comment:
Typo: check_t*r*ansaction

Reply:
Also, this method is only used by API calls and should therefore move into `app` library instead.

Reply:
will do this